session_0/fizz_buzz.rb

```diff
@@ -29,13 +29,27 @@
 # For example, fizz_buzz(n: 6, x: 2, y: 3) should return the following:
 # ["1", "Fizz", "Buzz", "Fizz", "5", "FizzBuzz"]
 #
-# What happens if the function is called by x (or y) = 0 or negative? 
+# What happens if the function is called by x (or y) = 0 or negative?
 #
 # The divisiblity by zero is not defined and neither do the little
 # children know how to divide by negative numbers.
 #
 # We will raise an `ArgumentError` exception to let the caller know that
 # their function arguments were incorrect.
 def fizz_buzz(n:, x: 3, y: 5)
-  raise NotImplementedError # TODO
+  raise ArgumentError if x <= 0 or y <= 0 or n < 0
```

Comment:
We prefer `&&` and `||` over `and` and `or` operators: https://rubystyle.guide/#no-and-or-or